app/src/assets/utilities/index.js

```
@@ -4,9 +4,13 @@
 
 const IMAGES_DIR = '../images/';
 const appImages = {
-  logo: require(IMAGES_DIR + 'logowithWBG.jpg'),
+  logo: require(IMAGES_DIR + 'logowithwhiteBG.png'),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)